include/RAJA/pattern/kernel/ForICount.hpp

```diff
@@ -112,7 +112,7 @@ struct StatementExecutor<
 
     auto r = resources::get_resource<ExecPolicy>::type::get_default();
 
-    forall_impl(r, ExecPolicy{}, TypedRangeSegment<len_t>(0, len), for_wrapper);
+    forall_impl(r, ExecPolicy{}, TypedRangeSegment<len_t>(0, len), for_wrapper, RAJA::expt::get_empty_forall_param_pack());
```

Comment:
It looks like this will work with ``RAJA::kernel`` as well right? do we want to add a functional test?

Reply:
This doesn't work with kernel but we need to pass an empty ForallParamPack so that it know not to call the experimental forall implementation.